pkgs/development/compilers/dtc/default.nix

```
@@ -22,6 +23,14 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-Zl2bOGDJIe/bhMFNsy969JYCYqtXTOzgb+bbOlKqOco=";
   };
 
+  patches = [
+    (fetchpatch {
+      # https://github.com/dgibson/dtc/pull/141
+      url = "https://github.com/dgibson/dtc/commit/56a7d0cb3be5f2f7604bc42299e24d13a39c72d8.patch";
+      hash = "sha256-LBbG/0c5HCaX6eU1gGz8BC5ZxeqeI9PW4PzpUxwYQJE=";
+    })
```

```suggestion
    (fetchpatch {
      # https://github.com/dgibson/dtc/pull/141
      url = "https://github.com/dgibson/dtc/commit/56a7d0cb3be5f2f7604bc42299e24d13a39c72d8.patch";
      hash = "sha256-GmAyk/K2OolH/Z8SsgwCcq3/GOlFuSpnVPr7jsy8Cs0=";
    })
```

`fetchpatch2` uses the updated version of patchutils, while `fetchpatch` uses patchutils 0.3.3 for hash compatibility.